cli/src/tests.rs

```diff
@@ -973,6 +973,73 @@ async fn setup_shamir(tmp_path: TmpPath) {
     // TODO list shamir setup
 }
 
+#[rstest::rstest]
+#[tokio::test]
+async fn workspace_import_file(tmp_path: TmpPath) {
+    let _ = env_logger::builder().is_test(true).try_init();
+    let tmp_path_str = tmp_path.to_str().unwrap();
+    let config = get_testenv_config();
+    let (url, [alice, ..], _) = run_local_organization(&tmp_path, None, config)
+        .await
+        .unwrap();
+
+    set_env(&tmp_path_str, &url);
```

Comment:
This function internally uses `std::env::set_var` so it sets environ variable for the whole process, this will cause random issue when multiple tests are run concurrently in the same process (so if we use `cargo test` instead of `cargo nextest`)

Instead it would be better to pass the environ variable to the `Command::cargo_bin` command (so that only this subprocess has those variable configured)

This is outside the scope of this PR (since `set_env` is used in each test), so can you create an issue about this ?

Reply:
https://github.com/Scille/parsec-cloud/issues/8079